### ISSUE TITLE
Escape secquences are discarded in description strings during introspection, resulting in invalid schema

### ISSUE THREAD

**Issue by kije** [jimkyndemeyer/js-graphql-intellij-plugin#346]
**Describe the bug**
When an introspection on an endpoint is performed, escape sequences in the description strings are "unescaped"/discarded in the resulting schema file. This leads to invalid schema files when there are description strings that use such escape sequences.

This happens, for example, when a description of a type/field/etc... contains a regex, as they often contain tokens like `\s`, `\d` etc..., where the backslash `\` needs to be escaped (`\\`) in order to be [valid in the description string](https://spec.graphql.org/June2018/#sec-Descriptions). 

Here an example:
Simplified introspection query
```graphql
query {
    __schema {
        types {
            name
            description
        }
    }
}

```

Query result:
```json5
{
    "data": {
        "__schema": {
            "types": [
                   // ...
                   {
                       "name": "YearMonth",
                       "description": "A field whose value matches the provided regular expression /^(\\d{4}-(0[1-9]|1[012]))$/."
                   },
                   // ...
            ]
        }
    }
}
```

Generated schema file by `js-graphql-intellij-plugin`
```graphql

"A field whose value matches the provided regular expression /^(\d{4}-(0[1-9]|1[012]))$/."
scalar YearMonth
```

**Expected behavior**
The plugin should not unescape/discard escape sequences from the introspection.

**Screenshots**
![image (4)](https://user-images.githubusercontent.com/3481639/84164362-b6e9f100-aa72-11ea-9991-d4862e1b5f85.png)

**Version and Environment Details**
Operation system: Linux 5.4.0-7626-generic
IDE name and version: WebStorm 2020.1.2 
Plugin version: 2.5.0

**Comment by jimkyndemeyer** [comment]
Hi Kim,

Thanks for using the plugin.

Can you try 2.4.0 from https://plugins.jetbrains.com/plugin/8097-js-graphql/versions to confirm whether this is regression in 2.5.0?

You can temporarily install 2.4.0 it using the gear icon in the plugins dialog and loading it from disk.

Best regards,
Jim

**Comment by kije** [comment]
Hi Jim

I just tested it with v2.4.0 of the plugin, and the escape sequences are correctly preserved, so it seems to be a regression in v2.5.0

![image](https://user-images.githubusercontent.com/3481639/84815831-e620d500-b013-11ea-9a54-8a4a9cfc120a.png)

Best regards,
Kim

**Comment by jimkyndemeyer** [comment]
Hi Kim,

I took another look at this, and I can't reproduce based on the JSON you've attached to the issue. This is what I see:

![image](https://user-images.githubusercontent.com/15837712/87665153-c392f080-c766-11ea-94df-57f8457ead5b.png)

There is also a test case that checks for the proper escaping, and it didn't fail after the graphql-java 15 upgrade. Do you have any other information that can help reproduce this?

**Comment by vepanimas** [comment]
Fixed in #359, only the scalar types were affected.
